Add --subset argument to predict command (#63)
Imageomics/pybioclip: docs/command-line-help.md

  -h, --help     show this help message and exit
  --model MODEL  list available tags for pretrained model checkpoint(s) for
                 specified model
```

> ## bioclip list-tol-taxa
> List taxa embedding labels included with the tree of life model as a CSV.
> ```
> usage: bioclip list-tol-taxa [-h]
> 
> options:
>   -h, --help  show this help message and exit

Reviewer: ```suggestion
## bioclip list-tol-taxa
List taxa embedding labels included with the tree of life model in CSV format in the CLI.
```
usage: bioclip list-tol-taxa [-h]

options:
  -h, --help  show this help message and exit
```
I think this could use a clearer instruction/sample usage (at least in the docs), like in your [comment about adding it](https://github.com/Imageomics/pybioclip/pull/63#issuecomment-2488839452).
As-is it may not be immediately apparent that it will print the entire thing in the terminal and that does require some other code to be used effectively.

Author: This text is from the arg parser so changes will be needed there as well:
https://github.com/Imageomics/pybioclip/blob/c767845dd2310f646f47931f725fcb3bca7c9171/src/bioclip/__main__.py#L137
